kitty-items-web/src/flow/fetch-market-item.script.js

```diff
@@ -0,0 +1,119 @@
+import * as fcl from "@onflow/fcl"
+import * as t from "@onflow/types"
+// import {batch} from "./util/batch"
```

Comment:
This is supposed to be commented out. The commented out code is the code we want to be using, but can't yet do to contract stuff.